server/issuers.go

```diff
@@ -214,6 +335,17 @@ func (c *Server) issuerRouterV2() chi.Router {
 	if os.Getenv("ENV") == "production" {
 		r.Use(middleware.SimpleTokenAuthorizedOnly)
 	}
+	r.Method("GET", "/{type}", middleware.InstrumentHandler("GetIssuerV2", handlers.AppHandler(c.issuerHandlerV2)))
```

Comment:
Apart from the instrumentation is there a difference with the route registered below

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/c212d77eac67c1cc0369b89c95e4dea5812fd3fe